apps/test/unit/applab/ApplabVisualizationColumnTest.js

```diff
@@ -28,7 +29,8 @@ describe('AppLabVisualizationColumn', () => {
           widgetMode
           isIframeEmbed
           playspacePhoneFrame
-        />
+        />,
+        options
```

Comment:
super nit: i think you could use an object literal here instead of creating an `options` variable

```javascript
shallow(
  <UnconnectedApplabVisualizationColumn />,
  {disableLifecycleMethods: true}
)
```

Reply:
Sure! Will do.